### PR TITLE
added possibility to either scale all dimensions with one factor or...

### PR DESCRIPTION
**Pull request by beschu**
…set scaling factors for each dimensions manually.

Issue:
 Image Resizer: Allow to set one dimension and scale the others automatically #340 

@dietzc @hornm @gab1one @tibuch 

I also updated the Image Resizer Test Workflow which i have ready to be uploaded after the pull request has been approved and merged.

### CODE REVIEW

**Comment by dietzc** [comment]
Thanks @beschu!

One question about the testflow: 
Will old nodes (this means nodes which have been created before the change)  still be loaded correctly? Did you change any executed nodes from the old testworkflow or you added just another flow for the new parameters?

**Comment by beschu** [comment]
Good question!
Where the Parameter tests are done i just added another Metanode called "Dim Manual vs Dim 1 Parameter". In this Metanode I have two Image Resizer nodes, one that scales all Dimensions with all the scaling factors set manually and the other one which uses only 1 scaling factor for all the Dimensions. Then i compared the two resulting images with an Image Comparator.

From the old workflow i didn't change any nodes. But if I open those Image Resizers i see my new added options ready to be used.

**Comment by dietzc** [comment]
ok, so can you open/load the old workflow without any complaints?

**Comment by beschu** [comment]
Okay actually I can execute the old workflow on the Community Server but there is a message "Problems occured while loading settings." on some of the Image Resizer nodes.
@tibuch told me it has to do with models the node now tries to load, it didn't have before and that I somehow need to catch this.

**Comment by dietzc** [comment]
fixed?

**Comment by beschu** [comment]
fixed! Should I wait until it is merged with uploading the test workflow to 3.2 and trunk or do it before?

File: org.knime.knip.base/src/org/knime/knip/base/nodes/proc/resizer/ResizerNodeFactory.java [hunk 1]
```diff
@@ -249,7 +249,6 @@ protected String getDefaultSuffixForAppend() {
 
             private final SettingsModelDouble m_inputFactorModel = createAffectAllDimensionModel();
 
-            //ADDED BY ME
```
Comment (dietzc) [inline org.knime.knip.base/src/org/knime/knip/base/nodes/proc/resizer/ResizerNodeFactory.java]: ?! whats that?

**Comment by dietzc** [comment]
you can upload the workflow now. thanks @beschu

File: org.knime.knip.base/src/org/knime/knip/base/nodes/proc/resizer/ResizerNodeFactory.java (hunk 1 above)
Comment (beschu) [inline org.knime.knip.base/src/org/knime/knip/base/nodes/proc/resizer/ResizerNodeFactory.java]: That was just an annotation in the code to see something I added, to be able to find it again easier. I forgot to delete it before the commit, so i deleted it

**Comment by beschu** [comment]
The workflows are now updated! @dietzc

**Pull request merged**